birdset/modules/multiclass_module.py

```diff
@@ -0,0 +1,58 @@
+import torch
```

Comment:
@jonathanschirp This is not needed. The `base_module` is for multiclass and we have the `multilabel_module` for multilabel.